### ISSUE TITLE
Case sensitive: does not contain a unique column

### ISSUE THREAD

**Issue by bm322741** [phpmyadmin/phpmyadmin#15346]
### Describe the bug

The check for primary key is case sensitive.

### To Reproduce

select id from product [works]
select ID from product [gives no unique column error]

### Expected behavior

Both should be unique columns for editing. Do a case-insensitive check.

### Screenshots

![image](https://user-images.githubusercontent.com/46461343/59937745-49881f00-9421-11e9-841b-3fc2e0a7f889.png)

### Server configuration

- PHP version: 7.3
- phpMyAdmin version: 4.8.4

**Comment by williamdes** [comment]
https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html

@bm322741 can you please send the output of:
```sql
SELECT @@lower_case_table_names;
```

**Comment by bm322741** [comment]
SELECT @@lower_case_table_names returns 0

**Comment by saurass** [comment]
@bm322741 can you please try to reproduce this error [here](https://demo.phpmyadmin.net/STABLE/tbl_sql.php?db=jsf_estudent&table=korisnici)

**Comment by williamdes** [comment]
```sql
CREATE TABLE `opengis`.`15346_not_unique` ( `ID` INT UNSIGNED NOT NULL AUTO_INCREMENT , `content` VARCHAR(255) NOT NULL , PRIMARY KEY (`ID`)) ENGINE = InnoDB COMMENT = 'Issue #15346 not unique';

INSERT INTO `15346_not_unique` (`ID`, `content`) VALUES (NULL, 'a'), (NULL, 'b');
INSERT INTO `15346_not_unique` (`ID`, `content`) VALUES (NULL, 'c'), (NULL, 'd');
```

Can you send more details about the database server (on the phpMyAdmin homepage) ?

**Comment by bm322741** [comment]
Is this all the info you need? In the previous comment I was asked to try on a test server but I don't have/couldn't find any credentials for it.
![image](https://user-images.githubusercontent.com/46461343/60679907-4d6f6480-9e57-11e9-8650-6f37a9e62a54.png)

**Comment by ShailuJain** [comment]
@williamdes is this issue solved? I can't reproduce this issue on the current master branch.

**Comment by williamdes** [comment]
@ShailuJain I think not, you should check the value of `SELECT @@lower_case_table_names;`

**Comment by ShailuJain** [comment]
On my local system `SELECT @@lower_case_table_names;` returns 1. Locally i am on the latest master branch.
![image](https://user-images.githubusercontent.com/35773853/69493198-f2a3db80-0ed1-11ea-9c1d-4b919f87dfdf.png)

An on https://demo.phpmyadmin.net/master-config/index.php?route=/server/sql  `SELECT @@lower_case_table_names;` returns 0
![image](https://user-images.githubusercontent.com/35773853/69493216-0e0ee680-0ed2-11ea-8a30-33015e1ccfa5.png)

**Comment by williamdes** [comment]
@ShailuJain using my test data you could not reproduce the issue on any of your systems or demo servers ?
https://github.com/phpmyadmin/phpmyadmin/issues/15346#issuecomment-505844813

**Comment by ShailuJain** [comment]
> @ShailuJain using my test data you could not reproduce the issue on any of your systems or demo servers ?
> [#15346 (comment)](https://github.com/phpmyadmin/phpmyadmin/issues/15346#issuecomment-505844813)

Yes reproduced by `SELECT ID, content FROM 15346_not_unique;` this query.

**Comment by tabraiz961** [comment]
SELECT @@lower_case_table_names; //returns 1
used your query to create table and insert data 

Could reproduce the error when using the query 
`SELECT ID, content FROM 15346_not_unique;`
i.e 15346_not_unique without single quotes
using single quotes with database name does not show the warning and fields are editable.
